### PR TITLE
Update intl_it.h

### PR DESCRIPTION
**Pull request by stewecar**
I modified a few not available voices of the italian configuration's page.
Thanks!
Cheers!

### CODE REVIEW

**Comment by dirkmueller** [comment]
There are more missing translations in line 98 and afterwards. Could you please fill them in as well?

**Comment by stewecar** [comment]
Done now..sorry i didn't notice them..

**Comment by dirkmueller** [comment]
@stewecar no problem. a few more:

```
fgrep '""' intl_it.h 
const char INTL_DNMS_CORRECTION[] PROGMEM ="";
const char INTL_FS_WIFI[] PROGMEM = "";
const char INTL_FS_WIFI_DESCRIPTION[] PROGMEM = "";
const char INTL_FS_WIFI_NAME[] PROGMEM = "";
const char INTL_TIME_SINCE_LAST_MEASUREMENT[] PROGMEM = "";
#define INTL_NUMBER_OF_MEASUREMENTS ""
#define INTL_TIME_SENDING_MS ""
```

File: airrohr-firmware/intl_it.h [hunk 1]
```diff
@@ -38,15 +38,15 @@ const char INTL_FS_WIFI_DESCRIPTION[] PROGMEM = "";
...
 const char INTL_DISPLAY[] PROGMEM = "OLED SSD1306";
 const char INTL_SH1106[] PROGMEM = "OLED SH1106";
-const char INTL_FLIP_DISPLAY[] PROGMEM = "";
+const char INTL_FLIP_DISPLAY[] PROGMEM = "Ruota il display OLED di 180°";
 const char INTL_LCD1602_27[] PROGMEM = "LCD 1602 (I2C: 0x27)";
 const char INTL_LCD1602_3F[] PROGMEM = "LCD 1602 (I2C: 0x3F)";
 const char INTL_LCD2004_27[] PROGMEM = "LCD 2004 (I2C: 0x27)";
-const char INTL_DISPLAY_WIFI_INFO[] PROGMEM = "";
-const char INTL_DISPLAY_DEVICE_INFO[] PROGMEM = "";
+const char INTL_DISPLAY_WIFI_INFO[] PROGMEM = "Visuallizza informazioni del wi-fi";
```
Comment (stewecar) [inline airrohr-firmware/intl_it.h]: "Visualizza" not "Visuallizza", thanks!

File: airrohr-firmware/intl_it.h [hunk 2]
```diff
@@ -38,15 +38,15 @@ const char INTL_FS_WIFI_DESCRIPTION[] PROGMEM = "";
...
+const char INTL_DISPLAY_DEVICE_INFO[] PROGMEM = "Visuallizza informazioni del dispositivo";
```
Comment (stewecar) [inline airrohr-firmware/intl_it.h]: "Visualizza" not "Visuallizza", thanks!

**Comment by stewecar** [comment]
@dirkmueller ..done now..thanks.

File: airrohr-firmware/intl_it.h (hunk 2 above)
Comment (dirkmueller) [reply airrohr-firmware/intl_it.h]: fixed in #565, thanks!

**Comment by stewecar** [comment]
Thanks to you!

Il mar 26 nov 2019, 21:26 Dirk Mueller <notifications@github.com> ha
scritto:

**Pull request merged**